SportsHub/UnitTests/Controllers/UserControllerTests.cs

```diff
@@ -80,10 +80,10 @@ public void AdminsEndPoint_Returning_OkStatus()
         _userController.ControllerContext = new ControllerContext();
         _userController.ControllerContext.HttpContext = new DefaultHttpContext() { User = user };
 
-        //Act
-        var result = _userController.AdminsEndpoint() as ObjectResult;
+        ////Act
+        //var result = _userController.AdminsEndpoint() as ObjectResult;
```

Comment:
If code is commented, I assume it is not used. Please remove it if you don't need it.